From 767cb99a3ed3b97fcff296a8c23139ba8dbded0c Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:26 +0900
Subject: [PATCH 0958/1083] Revert "drm/rcar-du: Enable the DE signal"

This reverts commit 9652c08db3d784f44d566f1b8346f18927fe0883.
(cherry picked from commit 12a651e883c07dbcf5ccad690d9b08405e256677)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_crtc.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_crtc.c b/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
index 4906a7f..ca80939 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
@@ -80,8 +80,7 @@ static void rcar_du_crtc_set_display_timing(struct rcar_du_crtc *rcrtc)
 
 	/* Signal polarities */
 	value = ((mode->flags & DRM_MODE_FLAG_PVSYNC) ? 0 : DSMR_VSL)
-	      | ((mode->flags & DRM_MODE_FLAG_PHSYNC) ? 0 : DSMR_HSL)
-	      | DSMR_DIPM_DE;
+	      | ((mode->flags & DRM_MODE_FLAG_PHSYNC) ? 0 : DSMR_HSL);
 	rcar_du_crtc_write(rcrtc, DSMR, value);
 
 	/* Display timings */
@@ -98,9 +97,6 @@ static void rcar_du_crtc_set_display_timing(struct rcar_du_crtc *rcrtc)
 	rcar_du_crtc_write(rcrtc, VSPR, mode->vtotal - mode->vsync_end +
 					mode->vsync_start - 1);
 	rcar_du_crtc_write(rcrtc, VCR,  mode->vtotal - 1);
-
-	rcar_du_crtc_write(rcrtc, DESR,  mode->htotal - mode->hsync_start);
-	rcar_du_crtc_write(rcrtc, DEWR,  mode->hdisplay);
 }
 
 static void rcar_du_crtc_start_stop(struct rcar_du_crtc *rcrtc, bool start)
-- 
1.8.3.2

